po/ru.po

```diff
@@ -1199,7 +1199,7 @@ msgid ""
 "This copy was built from %s rev %s on %s."
 msgstr ""
 "%s\n"
-"Эта копия собрана из %s, ревизия %s. Дата сборки %s."
+"Эта копия собрана из %s, ревизия %s. \nДата сборки %s."
```

Comment:
It is better to insert only layout elements, which are already in the msgid. You don't know the size of the displaying window.